sqlparser/seata-sqlparser-core/src/main/java/io/seata/sqlparser/struct/SqlDefaultExpr.java

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
why not use com.alibaba.druid.sql.ast.expr.SQLSequenceExpr

Reply:
sqlparser module can be multi implemention. druid just one of them.